test/test_implementation/process_dist.jl

```diff
@@ -1,18 +1,18 @@
 
-struct TestProcessDistribution{PROC,MODEL,PSDEF} <: ScatteringProcessDistribution
+struct TestProcessDistribution{PROC,MODEL,PSL} <: ScatteringProcessDistribution
```

Comment:
We should probably name these testing implementations mock in all our packages now and put them in a Mock submodule of their own.

Reply:
You are right, of course. I want to do this in a separate PR because it goes beyond the intended changes of this one. 

Furthermore, I think the renaming `Mock<struct>` of the test implementation and then moving to a `.mock` submodule should be done in `QEDbase` once the ParticleDistributions interface is moved there. 

I will open the respective issues.